django_netjsonconfig/tests/test_admin.py

```diff
@@ -400,3 +400,28 @@ def test_preview_device_with_context(self):
         }
         response = self.client.post(path, data)
         self.assertContains(response, "option interval &#39;60&#39;")
+
+    def test_context_device_without_config(self):
+        device = self._create_device(name='sample-device')
+        url = reverse('admin:django_netjsonconfig_device_context', args=[device.pk])
+        response = self.client.get(url)
+        self.assertContains(response, '"name": "sample-device"')
```

Comment:
this way of testing your code is not good enough.

Use response.json() and compare it to the result returned by get_context(), they must be equal.

Eg:

```python
self.assertEqual(response.json(), Config(device=device).get_context())
```